### PR TITLE
New enhancements

### PR DESCRIPTION
**Pull request by sanketsmali03**
- Added Maven and Gradle execution profiles 
- Added Allure Report integration for Maven
- Added Jenkins File and linked to the following [job](https://customer-demos.bsstag.com/view/Customer%20Demo%20Repositories/job/browserstack-examples-cucumber-testng/61/)
- Add capabilities yaml files

### CODE REVIEW

File: src/test/resources/capabilities.yml [hunk 1]
```diff
@@ -10,47 +10,13 @@ cloudDriver:
     enable: false
   common_capabilities:
     project: BrowserStack Demo Repository
-    buildPrefix: browserstack-examples-cucumber-testng
+    buildPrefix: browserstacktestng
```
Comment (anirudha13) [inline src/test/resources/capabilities.yml]: Keep it the same as the repo name. Easier to track.

File: src/test/resources/capabilities-parallel-browsers.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+driverType: cloudDriver
+
+cloudDriver:
+  hubUrl: https://hub-cloud.browserstack.com/wd/hub
+  user: BROWSERSTACK_USERNAME
+  accessKey: BROWSERSTACK_ACCESSKEY
+  common_capabilities:
+    project: BrowserStack Demo Repository
+    buildPrefix: browserstack-examples-testng
```
Comment (anirudha13) [inline src/test/resources/capabilities-parallel-browsers.yml]: change to **browserstack-examples-cucumber-testng**

File: src/test/resources/capabilities-local.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  localTunnel:
+    enabled: true
+    local_options:
+      forcelocal: true
+      force: true
+      verbose: true
+      logFile: local_test.log
+  common_capabilities:
+    project: BrowserStack Demo Repository
+    buildPrefix: browserstack-examples-testng
```
Comment (anirudha13) [inline src/test/resources/capabilities-local.yml]: change to **browserstack-examples-cucumber-testng**

File: src/test/resources/capabilities-local-parallel.yml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  localTunnel:
+    enabled: true
+    local_options:
+      forcelocal: true
+      force: true
+      verbose: true
+      logFile: local_test.log
+  common_capabilities:
+    project: BrowserStack Demo Repository
+    buildPrefix: browserstack-examples-testng
```
Comment (anirudha13) [inline src/test/resources/capabilities-local-parallel.yml]: change to **browserstack-examples-cucumber-testng**

File: src/test/resources/capabilities-local-parallel-browsers.yml [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+  localTunnel:
+    enabled: true
+    local_options:
+      forcelocal: true
+      force: true
+      verbose: true
+      logFile: local_test.log
+  common_capabilities:
+    project: BrowserStack Demo Repository
+    buildPrefix: browserstack-examples-testng
```
Comment (anirudha13) [inline src/test/resources/capabilities-local-parallel-browsers.yml]: change to **browserstack-examples-cucumber-testng**

File: src/test/resources/capabilities-docker-parallel-browsers.yml [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+  localTunnel:
+    enable: true
+    local_options:
+      forcelocal: true
+      force: true
+      verbose: true
+      logFile: local_test.log
+  common_capabilities:
+    project: BrowserStack Demo Repository
+    buildPrefix: browserstack-examples-junit5
```
Comment (anirudha13) [inline src/test/resources/capabilities-docker-parallel-browsers.yml]: change to **browserstack-examples-cucumber-testng**

**Review by anirudha13 — CHANGES_REQUESTED** [review]

File: src/test/resources/capabilities.yml (hunk 1 above)
Comment (sanketsmali03) [reply src/test/resources/capabilities.yml]: Corrected the Build Prefix name to browserstack-examples-cucumber-testng

File: src/test/resources/capabilities-parallel-browsers.yml (hunk 2 above)
Comment (sanketsmali03) [reply src/test/resources/capabilities-parallel-browsers.yml]: Corrected the Build Prefix name to browserstack-examples-cucumber-testng

File: src/test/resources/capabilities-local.yml (hunk 3 above)
Comment (sanketsmali03) [reply src/test/resources/capabilities-local.yml]: Corrected the Build Prefix name to browserstack-examples-cucumber-testng

File: src/test/resources/capabilities-local-parallel.yml (hunk 4 above)
Comment (sanketsmali03) [reply src/test/resources/capabilities-local-parallel.yml]: Corrected the Build Prefix name to browserstack-examples-cucumber-testng

File: src/test/resources/capabilities-local-parallel-browsers.yml (hunk 5 above)
Comment (sanketsmali03) [reply src/test/resources/capabilities-local-parallel-browsers.yml]: Corrected the Build Prefix name to browserstack-examples-cucumber-testng

File: src/test/resources/capabilities-docker-parallel-browsers.yml (hunk 6 above)
Comment (sanketsmali03) [reply src/test/resources/capabilities-docker-parallel-browsers.yml]: Corrected the Build Prefix name to browserstack-examples-cucumber-testng

**Review by anirudha13 — APPROVED** [review]

**Pull request merged**